sdcm/cluster.py

```diff
@@ -1304,10 +1304,9 @@ def datacenter_setup(self, datacenters):
         self.remoter.run(cmd)
 
     def config_setup(self, seed_address=None, cluster_name=None, enable_exp=True, endpoint_snitch=None,
-                     yaml_file=SCYLLA_YAML_PATH, broadcast=None, authenticator=None,
-                     server_encrypt=None, client_encrypt=None, append_conf=None, append_scylla_args=None,
-                     debug_install=False, hinted_handoff_disabled=False, murmur3_partitioner_ignore_msb_bits=None,
-                     authorizer=None):
+                     yaml_file=SCYLLA_YAML_PATH, broadcast=None, authenticator=None, server_encrypt=None,
+                     client_encrypt=None, append_conf=None, append_scylla_args=None, debug_install=False,
+                     hinted_handoff='enabled', murmur3_partitioner_ignore_msb_bits=None, authorizer=None):
```

Comment:
why did you choese to work with strings instead of boolean? it is error-prone
and if you do so, you need to validate the allowed values
 I would rename to `hinted_handoff_enabled`, since it is called so in the `scylla.yaml`

Reply:
It was requerement of @roydahan  (https://trello.com/c/IHMNsYSu/1196-hintedhandoffenabled-parameter-is-not-configured-by-sct-hintedhandoffdisabled-1182).
I suggested to rename to ```hinted_handoff_enabled``` and that remain it as boolean, but Roy didn't agree